src/core/shared/interfaces/Character.ts

```diff
@@ -1,4 +1,4 @@
-import { Vector3 } from 'alt-server';
+import { Vector3 } from 'alt-shared';
```

Comment:
Should use shared Vector from interfaces in `shared` folder.

Reply:
Yes must be ur interface tnx.